reference/promise-types.markdown

```diff
@@ -113,7 +113,30 @@ A "meta" attribute can likewise be added into any body (of any type).
     }
 ```
 
-**Note:** When a variable is re-defined the associated meta attributes are also
+Another example:
+
+```cf3
+    some_promise_type:
+      any::
+        "my_promiser"
+          meta => { "Team Foo", "workaround", "non-critical" };
+```
+
+The meta tags may be referred to programmatically in various ways, or may be solely for
+human consumption.  Meta tags on vars promises and classes promises are
+particularly suited for programmatic interpretation; meta tags on other
+promise types (or in bodies) are more likely to be intended only for human consumption.
+
+Relevant CFEngine functions are: `classesmatching()`,
+`classmatch()`, `countclassesmatching()`, `getclassmetatags()`,
+`getvariablemetatags()`, `variablesmatching()`, `variablesmatching_as_data()`
```

Comment:
Are these going to be rendered as links to the particular functions' documentation?

Reply:
I believe so.  I imitated other parts of the documentation where they are so rendered.